lib/helpers/constants.py

```diff
@@ -0,0 +1,2 @@
+BASE_URL = "https://api.checkr-staging.com/v1"
+STAGING_URL = "https://api.checkr-staging.com/v1"
```

Comment:
I don't actually know how to address this part since it's an open source client.

Reply:
Pull it from the env and if it isn't present default to a default staging url.